lib/Sema/TypeCheckConcurrency.h

```diff
@@ -467,6 +467,8 @@ bool isPotentiallyIsolatedActor(
     VarDecl *var, llvm::function_ref<bool(ParamDecl *)> isIsolated =
                       [](ParamDecl *P) { return P->isIsolated(); });
 
+Optional<AttributedImport<ImportedModule>> findImportFor(
+    ModuleDecl *nominalModule, ASTContext *nominalASTCtx, const DeclContext *fromDC);
```

Comment:
I still think this would probably be better off as a method on `DeclContext`. Tho if you want to keep it as a top-level function, could you move it out of TypeCheckConcurrency.h + .cpp? ImportCache.h + .cpp seems like it might be a better place for such a top-level function.

Reply:
Moved to DeclContext